### PR TITLE
Event equips and NPC names.

### PR DESCRIPTION
**Pull request by zko-jin**

### CODE REVIEW

File: ui/00_message/common/item_name.xml [hunk 1]
```diff
@@ -29434,243 +29434,323 @@
...
     <original>レウスヘルム</original>
+    <translation>Rathalos Helm</translation>
   </item>
   <item id="10907">
     <original>レウスヘルム</original>
+    <translation>Rathalos Helm</translation>
   </item>
   <item id="10908">
     <original>レイアヘルム</original>
+    <translation>Rathalos Helm</translation>
```
Comment (riftcrystal) [inline ui/00_message/common/item_name.xml:29457]: This is 'Rathian', not 'Rathalos'.

File: ui/00_message/common/item_name.xml [hunk 2]
```diff
@@ -29434,243 +29434,323 @@
...
   </item>
   <item id="10909">
     <original>レイアヘルム</original>
+    <translation>Rathalos Helm</translation>
```
Comment (riftcrystal) [inline ui/00_message/common/item_name.xml:29461]: This is 'Rathian', not 'Rathalos'.

File: ui/00_message/common/item_name.xml [hunk 3]
```diff
@@ -29434,243 +29434,323 @@
...
   </item>
   <item id="10910">
     <original>レイアヘルム</original>
+    <translation>Rathalos Helm</translation>
```
Comment (riftcrystal) [inline ui/00_message/common/item_name.xml:29465]: This is 'Rathian', not 'Rathalos'.

File: ui/00_message/common/item_name.xml [hunk 4]
```diff
@@ -29434,243 +29434,323 @@
...
   </item>
   <item id="10911">
     <original>レイアヘルム</original>
+    <translation>Rathalos Helm</translation>
```
Comment (riftcrystal) [inline ui/00_message/common/item_name.xml:29469]: This is 'Rathian', not 'Rathalos'.

File: ui/00_message/common/item_name.xml [hunk 5]
```diff
@@ -29434,243 +29434,323 @@
...
   </item>
   <item id="10912">
     <original>レイアヘルム</original>
+    <translation>Rathalos Helm</translation>
```
Comment (riftcrystal) [inline ui/00_message/common/item_name.xml:29473]: This is 'Rathian', not 'Rathalos'.

File: ui/00_message/npc/func_class_name.xml [hunk 6]
```diff
@@ -165,6 +165,7 @@
   <message>
     <key>FUNC_CLASS_NAME_39</key>
     <original>エクストリームミッション</original>
+    <translation>Extreme Mission</translation>
```
Comment (riftcrystal) [inline ui/00_message/npc/func_class_name.xml]: This should probably be plural.

File: ui/00_message/npc/func_class_name.xml [hunk 7]
```diff
@@ -218,6 +219,7 @@
   <message>
     <key>FUNC_CLASS_NAME_51</key>
     <original>アチーブメント</original>
+    <translation>Achievement</translation>
```
Comment (riftcrystal) [inline ui/00_message/npc/func_class_name.xml]: This should probably be plural.

File: ui/00_message/npc/npc_name.xml [hunk 8]
```diff
@@ -2123,6 +2124,7 @@
   <message>
     <key>NPC_NAME_731</key>
     <original>エウニケ</original>
+    <translation>Eunik</translation>
```
Comment (riftcrystal) [inline ui/00_message/npc/npc_name.xml]: This is 'Eunike'.

File: ui/00_message/npc/npc_name.xml [hunk 9]
```diff
@@ -2467,6 +2472,7 @@
   <message>
     <key>NPC_NAME_831</key>
     <original>ロイグ</original>
+    <translation>Loig</translation>
```
Comment (riftcrystal) [inline ui/00_message/npc/npc_name.xml]: This is 'Loeg'.

**Comment by riftcrystal** [comment]
Good looking changes, if you can correct the issues I can merge this in.

File: ui/00_message/npc/npc_name.xml [hunk 10]
```diff
@@ -2475,11 +2481,12 @@
   <message>
     <key>NPC_NAME_838</key>
     <original>リュカ</original>
+    <translation>Luka</translation>
   </message>
   <message>
     <key>NPC_NAME_839</key>
     <original>ベルトラン</original>
-    <translation>Bertrand</translation>
+    <translation>Betland</translation>
```
Comment (riftcrystal) [inline ui/00_message/npc/npc_name.xml]: This is actually Bertrand, please remove this change.

**Comment by zko-jin** [comment]
Fixed

**Pull request merged**